src/applications/personalization/dashboard/components/DashboardAlert.jsx

```diff
@@ -30,15 +30,23 @@ const DashboardAlert = ({
   return (
     <div className={classes}>
       <header>
-        {subheadline && <h4>{subheadline}</h4>}
-        <h3>{headline}</h3>
+        {subheadline && (
```

Comment:
I ran this locally to take a quick look at the `DashboardAlert` with the new markup and it all looks good to me.